core/src/dev/protection.c

```diff
@@ -158,8 +158,8 @@ Result expose_wallet(EncryptedMasterSeed *encrypted_master_seed,
     return Result_EXPOSE_WALLET_UNEXPECTED_MASTER_SEED_LEN_FAILURE;
   }
 
-  printf("master_seed: ");
-  for (unsigned int i = 0; i < r; i++) {
```

Comment:
Do you have an example of the bad output? What was the value of `(unsigned int) r`?

Reply:
`r` is the error code: 0 when no error occurs. Therefore, nothing is printed when it's used as the bound variable.